packages/sdk-core/src/modules/converter/StackTraceConverter.ts

```diff
@@ -0,0 +1,161 @@
+        const line = parseInt(sourceCodeParts[sourceCodeParts.length - 2]);
+        const library = sourceCodeParts.slice(0, sourceCodeParts.length - 2).join(':');
+        return {
+            funcName: functionName,
+            column: isNaN(column) ? undefined : column,
+            line: isNaN(line) ? undefined : line,
+            library: library,
+        };
+    }
+
+    private parseFirefoxStackFrame(stackFrame: string): BacktraceStackFrame {
+        const functionSeparator = '@';
+        const functionSeparatorIndex = stackFrame.indexOf(functionSeparator);
+        if (functionSeparatorIndex === -1) {
+            return {
+                funcName: stackFrame,
+                library: this.UNKNOWN_FRAME,
+            };
+        }
+
+        const functionName = stackFrame.substring(0, functionSeparatorIndex);
+        const sourceCodeInformation = stackFrame.substring(functionSeparatorIndex + functionSeparator.length);
+        const sourceCodeParts = sourceCodeInformation.split(':');
+
+        // check if the column information is available - if is not, we should use the legacy parser
+        const sourceCodeOrRowNumber = sourceCodeParts[sourceCodeParts.length - 2];
+        const possibleRowNumber = parseInt(sourceCodeOrRowNumber);
+
+        return isNaN(possibleRowNumber)
+            ? this.generateLegacyFirefoxFrame(functionName, sourceCodeParts)
+            : this.generateFirefoxFrame(functionName, sourceCodeParts);
+    }
+
+    private generateLegacyFirefoxFrame(functionName: string, sourceCodeParts: string[]): BacktraceStackFrame {
+        const lineNumberStart = sourceCodeParts.length - 1;
+        const lineNumber = parseInt(sourceCodeParts[lineNumberStart]);
+        const library = sourceCodeParts.slice(0, lineNumberStart).join(':');
+        return {
+            library,
+            funcName: functionName,
+            line: isNaN(lineNumber) ? undefined : lineNumber,
+        };
+    }
+
+    private generateFirefoxFrame(functionName: string, sourceCodeParts: string[]): BacktraceStackFrame {
+        const lineNumberStart = sourceCodeParts.length - 2;
+        const lineNumber = parseInt(sourceCodeParts[lineNumberStart]);
+        const columnName = parseInt(sourceCodeParts[lineNumberStart + 1]);
+        const library = sourceCodeParts.slice(0, lineNumberStart).join(':');
+
+        return {
+            library,
+            funcName: functionName,
+            column: columnName,
+            line: lineNumber,
+        };
+    }
+
+    private parseV8StackFrame(stackFrame: string): BacktraceStackFrame {
+        const frameSeparator = 'at ';
+        if (!stackFrame.startsWith(frameSeparator)) {
+            return {
+                funcName: stackFrame,
+                library: this.UNKNOWN_FRAME,
+            };
+        }
+
+        stackFrame = stackFrame.substring(stackFrame.indexOf(frameSeparator) + frameSeparator.length);
+        const sourceCodeSeparator = ' (';
```

Comment:
The separator does not appear with anonymous functions:

```
Error: lambdaAnonymousTest
    at http://localhost:8080/index.js:5:23 <-- anonymous
    at http://localhost:8080/index.js:6:15 <-- anonymous
    at http://localhost:8080/index.js:6:18 <-- anonymous
    at lambdaAnonymousTest (http://localhost:8080/index.js:6:21)
    at test (http://localhost:8080/index.js:52:9)
    at http://localhost:8080/index.js:58:1 <-- anonymous
```

Reply:
done